python/tvm/relax/frontend/onnx/onnx_frontend.py

```diff
@@ -430,7 +521,11 @@ class Conv(OnnxOpConverter):
 
     @classmethod
     def _impl_v11(cls, bb, inputs, attr):
-        ndim = len(inputs[0].struct_info.shape)
+        if hasattr(inputs[0].struct_info, "ndim"):
+            ndim = inputs[0].struct_info.ndim
+        else:
+            ndim = len(inputs[0].struct_info.shape)
```

Comment:
Is it possible that ndim is not avaible but shape is available?

Reply:
Yeah i was surprised to find this too, in the T5 model we end up with a tensor that has a defined shape but ndim=None. Im not sure why it happens though.